util/dbutil/variable.go

```diff
@@ -153,3 +154,145 @@ func ShowGrants(ctx context.Context, db QueryExecutor, user, host string) ([]str
 
 	return readGrantsFunc()
 }
+
+// CheckPrivilege uses the output of ShowGrants, to check `lackPriv` has been granted. Elements in `lackPriv` is deleted
+// during CheckPrivilege, and after it returns, `lackPriv` is the privileges that are not found in SHOW GRANTS.
+// level of keys of `lackPriv`: privilege => schema => table.
+func CheckPrivilege(grants []string, lackPriv map[mysql.PrivilegeType]map[string]map[string]struct{}) error {
```

Comment:
copied from https://github.com/pingcap/tiflow/blob/bd0cc36dd74d12aa7a71a6a6fd7e9d51f14eb242/dm/pkg/checker/privilege.go#L147

Reply:
Add this info in the comment?

Reply:
I'll remove the function in tiflow, let tiflow use this function

Reply:
> I'll remove the function in tiflow, let tiflow use this function

okay